crates/sargon/src/system/sargon_os/sargon_os_signing.rs

```diff
@@ -106,27 +97,25 @@ mod test {
             "https://example.com",
             DappDefinitionAddress::sample(),
         );
+        let auth_intent = AuthIntent::new_from_request(
+            nonce,
+            metadata,
+            vec![AddressOfAccountOrPersona::Account(account.address)],
```

Comment:
`vec!` can be removed if: https://github.com/radixdlt/sargon/pull/310/files#r1891586418